### DISCUSSION TITLE
OpenGLView on UAP

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#170] [General]
There is OpenGLView in MAUI . Will it work on UAP, which is Windows?

**Comment by Paegasus** [comment]
I tried it out today, it doesn't seem to work, I made a simple ContentVIew control with an ContentView,
I got the following runtime error.

"Catastrophic failure (0x8000FFFF (E_UNEXPECTED))"} | System.Exception {System.Runtime.InteropServices.COMException}

"Microsoft.UI.Xaml.Controls.Frame.NavigationFailed was unhandled."
